doc/developers.rst

```diff
@@ -42,8 +42,8 @@ You should see this:
 
   upstream	  https://github.com/glm-tools/pyglmnet.git (fetch)
   upstream	  https://github.com/glm-tools/pyglmnet.git (push)
-  origin	http://github.com/user/pyglmnet.git (fetch)
-  origin	http://github.com/user/pyglmnet.git (push)
+  origin	http://github.com/Your_User_Name/pyglmnet.git (fetch)
+  origin	http://github.com/Your_User_Name/pyglmnet.git (push)
```

Comment:
can you make this match the lines above?

Reply:
OK, done.